### PR TITLE
SUMO-239629: Updated aws api gateway app

### PR DESCRIPTION
**Pull request by shivani-sumo**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/amazon-aws/api-gateway.md [hunk 1]
```diff
@@ -322,8 +322,12 @@ Call the [UpdateStage](https://docs.aws.amazon.com/apigatewayv2/latest/api-refer
...
-3. Follow below steps for enabling access logs for each respective API type:
-   * Enable Access logs for REST APIs by referring to [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-logging-to-kinesis.html#set-up-kinesis-access-logging-using-console) and  when you specify the `Log format` field use the below JSON.
+3. Create a log group in CloudWatch Logs by referring to [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html). (Make sure to have log group name convention as `/aws/apigateway/<apiid>/<stagename>`)
+4. Follow below steps for enabling access logs for each respective API type:
+   :::note
+   Make sure to remove remove :* from the end while adding Access log destination ARN.
+   :::
+   * Enable Access logs for REST APIs by referring to [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html) and  when you specify the `Log format` field use the below JSON.
```
Comment (himanshu219) [inline docs/integrations/amazon-aws/api-gateway.md:330]: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html#set-up-access-logging-using-console lets use this link

**Review by himanshu219 — APPROVED** [review]
LGTM just a minor link change

File: docs/integrations/amazon-aws/api-gateway.md (hunk 1 above)
Comment (kimsauce) [inline docs/integrations/amazon-aws/api-gateway.md:330]:
```suggestion
3. Create a log group in CloudWatch Logs by referring to the [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html). Make sure to have your log group name convention as `/aws/apigateway/<apiid>/<stagename>`.
4. Enable Access logs for each respective REST API type by referring to the [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html#set-up-access-logging-using-console). When you specify the `Log format` field, use the below JSON.
   :::note
   Make sure to remove remove `:*` from the end while adding the Access log destination ARN.
   :::
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Suggestions for simplifying content

File: docs/integrations/amazon-aws/api-gateway.md (hunk 1 above)
Comment (shivani-sumo) [reply docs/integrations/amazon-aws/api-gateway.md:330]: Here under the 4th point, we have written steps for 3 different API types(REST API, HTTP API, and WebSocket API). So we cannot make REST API's steps direct as 4th point.

File: docs/integrations/amazon-aws/api-gateway.md [hunk 2]
```diff
@@ -322,8 +322,12 @@ Call the [UpdateStage](https://docs.aws.amazon.com/apigatewayv2/latest/api-refer
...
-3. Follow below steps for enabling access logs for each respective API type:
-   * Enable Access logs for REST APIs by referring to [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-logging-to-kinesis.html#set-up-kinesis-access-logging-using-console) and  when you specify the `Log format` field use the below JSON.
+3. Create a log group in CloudWatch Logs by referring to [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html). (Make sure to have log group name convention as `/aws/apigateway/<apiid>/<stagename>`)
+4. Follow below steps for enabling access logs for each respective API type:
+   :::note
+   Make sure to remove remove :* from the end while adding Access log destination ARN.
+   :::
+   * Enable Access logs for REST APIs by referring to [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html#set-up-access-logging-using-console) and  when you specify the `Log format` field use the below JSON.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/api-gateway.md:330]:
```suggestion
4. Follow the below steps to enable access logs for each respective API type:
   :::note
   Make sure to remove remove `:*` from the end while adding Access log destination ARN.
   :::
   * Enable Access logs for REST APIs by referring to the [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html#set-up-access-logging-using-console). When you specify the `Log format` field, use the below JSON.
```

File: docs/integrations/amazon-aws/api-gateway.md (hunk 1 above)
Comment (kimsauce) [reply docs/integrations/amazon-aws/api-gateway.md]: ah, I missed those other steps! fixed

**Review by kimsauce — APPROVED** [review]

File: docs/integrations/amazon-aws/api-gateway.md [hunk 3]
```diff
@@ -322,8 +322,12 @@ Call the [UpdateStage](https://docs.aws.amazon.com/apigatewayv2/latest/api-refer
    10. Click **Save**.
    11. Save the given URL of the source for next step.
 2. [Create Stack](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/#cloudformation-template) in AWS console with given CloudFormation Template.
-3. Follow below steps for enabling access logs for each respective API type:
-   * Enable Access logs for REST APIs by referring to [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-logging-to-kinesis.html#set-up-kinesis-access-logging-using-console) and  when you specify the `Log format` field use the below JSON.
+3. Create a log group in CloudWatch Logs by referring to [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html). (Make sure to have log group name convention as `/aws/apigateway/<apiid>/<stagename>`)
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/api-gateway.md:325]:
```suggestion
3. Create a log group in CloudWatch Logs by referring to the [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html). Make sure to set your log group name convention as `/aws/apigateway/<apiid>/<stagename>`.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**